test/performance/benchmarks/broker-imc/100-broker-perf-setup.yaml

```diff
@@ -24,35 +24,65 @@ kind: ConfigMap
 metadata:
   name: config-mako
   namespace: perf-eventing
-
 data:
-  _example: |
```

Comment:
Why remove these example blocks? Are they no longer relevant?

Reply:
I thought they are not relevant anymore, or am i wrong? :smile:

Reply:
Logically it makes no difference to keep them or not, but since they are still relevant, it would be better to keep them, just for reference :)